sources/Inputs/include/Configuration.h

```diff
@@ -184,6 +219,7 @@ class Configuration {
   boost::optional<double> precision_;                                                ///< Precision of the simulation
   double timeStep_ = 10.;                                                            ///< maximum value of the solver timestep
   double timeOfEvent_ = 10.;                                                         ///< time for contingency simulation (security analysis only)
+  std::map<ChosenOutputEnum, bool> chosenOutputs_;                                   ///< chosen configuration outputs
```

Comment:
For performance reasons please use a std::unordered_set and add the enum only if should be generated